airflow/docker.go

```diff
@@ -598,13 +598,6 @@ func (d *DockerCompose) UpgradeTest(newAirflowVersion, deploymentID, newImageNam
 	}
 	newDockerFile := destFolder + "/Dockerfile"
 
-	// check for dependency conflicts
-	if conflictTest {
-		err = d.conflictTest(testHomeDirectory, newImageName, newAirflowVersion)
-		if err != nil {
-			return err
-		}
-	}
```

Comment:
Nowhere in our code base do we seem to set `conflictTest` to `true`. There should be no way for this branch to ever run. This `conflictTest` used to exist as a flag on the CLI commands, but seems to have been removed.